requirements/dev.in

```diff
@@ -2,11 +2,13 @@
 -c test.txt
 -c docs.txt
 
+# If new packages are added here, check they don't end up in the PyInstaller
+# bundle. If they do, add to Analysis().excludes in /kart.spec
```

Comment:
> check they don't end up in the PyInstaller bundle

how? can you add some brief instructions or a link to docs on how to do this?

Reply:
Files being added are printed out during Pyinstaller bundle step